src/api/app/services/event_receivers_gatherer.rb

```diff
@@ -0,0 +1,33 @@
+# Gather event receivers for a receiver role
+#   An event receiver is a user/group wanting to be notified about an event
+#   A receiver role is defined on the event itself. It could be a maintainer, bugowner, etc...
+class EventReceiversGatherer
```

Comment:
Let's use `Fetcher` instead of `Gatherer`. There's already another service with that suffix. It will be consistent.

Reply:
Fixed